### ISSUE TITLE
Will this work with react hooks

### ISSUE THREAD

**Issue by iqbal125** [intljusticemission/react-big-calendar#1422]
#### What's the current behavior?

Will this library work with the latest version of react and hooks?

**Comment by TrySound** [comment]
Yes. Why shouldn't it? React didn't make any breaking changes while introducing hooks.

**Comment by iqbal125** [comment]
Sorry I phrased this incorrectly, I meant will Calendar this work _inside_ a react hooks component not alongside react hooks. 

Im asking because Im trying to do this right now and its not working, The class version is working but not the React hooks version. I thought Id ask before taking the time to create a detailed PR.

**Comment by TrySound** [comment]
Yes, hooks and classes are compatible. What exactly does not work for you? I can't help without any information. Provide https://codesandbox.io.
